action.yml

```diff
@@ -15,7 +15,12 @@ inputs:
   USERNAME:
     description: 'Your GitHub username'
     default: ${{ github.repository_owner }}
+    required: true
 
+  WEEK_FLAG:
```

Comment:
Why this is `WEEK_FLAG` here and `HEAD_FLAG` in the code?

Reply:
Lol, I was in a rush 😂

Reply:
😄  hehhe chill bro